vllm/v1/core/sched/scheduler.py

```diff
@@ -744,3 +743,17 @@ def make_stats(
             prefix_cache_stats=self.kv_cache_manager.make_prefix_cache_stats(),
             spec_decoding_stats=spec_decoding_stats,
         )
+
+    def make_spec_decoding_stats(
```

Comment:
could this be a separate util function instead of a method?  I think it would be nice if clutter was kept to a very bare minimum on scheduler.py

Reply:
Yeah, I considered it, but it pairs nicely with Scheduler `make_stats()` and it needs access to Scheduler `log_stats`